R/axis-secondary.R

```diff
@@ -236,8 +236,8 @@ AxisSecondary <- ggproto("AxisSecondary", NULL,
 
     # patch for date and datetime scales just to maintain functionality
     # works only for linear secondary transforms that respect the time or date transform
-    if (transformer$name %in% c("date", "time")) {
-      temp_scale <- self$create_scale(new_range, trans = transformer)
+    if (transformation$name %in% c("date", "time")) {
+      temp_scale <- self$create_scale(new_range, trans = transformation)
```

Comment:
We should probably change the name of the `trans` argument here as well, no?

Reply:
Yeah might as well do that too